gophpserialize.go

```diff
@@ -224,7 +230,7 @@ func (s *Serializer) move() {
 func Unmarshal(data []byte) (rv interface{}, err error) {
 	defer func() {
 		if r := recover(); r != nil {
-			err = fmt.Errorf("Cant Unmarshal data %v, error: %v", data, r)
+			err = fmt.Errorf("Cant Unmarshal data, error: %v", r)
```

Comment:
same here?

Reply:
Addressed in https://github.com/zalora/gophpserialize/pull/2/commits/d9e6a478889fe4a793221fe68da75b0498fa5024